### ISSUE TITLE
Fix the Readme File

### ISSUE THREAD

**Issue by 1point618** [1point618/exemplar-project-nikolai#3]
The `readme.md` file is a file that sits in the main directory of a repo. It's another way we use to communicate with each other. The `readme.md` file should include everything we need to get the repo running on our local computers. 

I've included the text we want in the readme below. Add the text, commit it to a new branch (**this is very important!**), then submit a pull request for the changes.

Once you've finished this lesson, I'll create a new `learning` issue and assign it to you, you'll need to go there to learn more.

---

If you need help, just @-mention me below asking for help and I'll send you a more detailed tutorial for how to do this. Again, keeping this issue open in its own tab while you do the work will make getting feedback and looking at the task list easier.
## Progress
- [x] Edit readme file to include the information below.
- [x] Commit the file to a new branch called "readme-bugfix".
- [x] Create a pull request, include the text `fixes #3`.
- [x] Once I comment with a :+1:, merge the PR.
- [x] Delete the branch.

**Comment by 1point618** [comment]
```
This is the content of the readme file.

* Right now this is just a blank repo.
* As we build more lessons, there will be more information in it.
* But every repo needs a readme file, so this is ours.
```

**Comment by chowdhury** [comment]
hey bossman @1point618 i need help

**Comment by 1point618** [comment]
Sure thing @chowdhury! I wanted to push you because you'd learned the rest so quickly, but this trips everyone up the first time they try it. What you need to do is follow the instructions below.
1. Click on the repo name.  
   ![Repo Name](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/repo-name.png)
2. Click on the readme.md file.  
   ![Readme File](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/readme-file.png)
3. Click on the edit button (it's a pencil).  
   ![Edit Icon](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/edit-icon.png)
4. Copy and paste the above text into the edit screen.  
   ![Edit Screen](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/edit-screen.png)
5. Add a commit title and message, or just use the default ones.
6. Click on the "create a new branch" button.
7. Name the branch `readme-bugfix`.  
   ![Create a New Branch](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/new-branch.png)
8. Propose the file change. This will take you to a new PR screen.
9. In the comment section of the PR, include the text `fixes #3`. This tells the PR that it is associated with this issue.
10. Create the pull request.  
    ![Create PR](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/create-pr.png)
11. Wait for the teacher to comment on your PR. 
12. Once they've given you the thumbs up, click "merge the pull request", and confirm the merge.  
    ![Merge PR](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/merge-pr.png)
13. Delete the branch.  
    ![Delete Branch](https://raw.githubusercontent.com/1point618/codename-exemplar/master/img/delete-branch.png)
14. Come back to this issue by clicking on the "#3" in the comment from earlier. This issue should be closed, because of the `fixes` tag.
15. Confirm that the issue is closed and all the boxes are checked above, then go to the Issues View and look for the new `leanring`-labeled issue I've created and assigned to you. 

Once you do, I'll tell you a bit more about _why_ we do things this way. It's called the GitHub Workflow and it's a pretty important part of how we work.
